optuna_dashboard/ts/components/TrialList.tsx

```diff
@@ -412,65 +423,91 @@ export const TrialList: FC<{ studyDetail: StudyDetail | null }> = ({
             </Menu>
           </ListSubheader>
           <Divider />
-          {trials.map((trial) => {
-            return (
-              <ListItem key={trial.trial_id} disablePadding>
-                <ListItemButton
-                  onClick={(e) => {
-                    if (e.shiftKey) {
-                      let next: number[]
-                      const selectedNumbers = selected.map((t) => t.number)
-                      const alreadySelected =
-                        selectedNumbers.findIndex((n) => n === trial.number) >=
-                        0
-                      if (alreadySelected) {
-                        next = selectedNumbers.filter((n) => n !== trial.number)
-                      } else {
-                        next = [...selectedNumbers, trial.number]
-                      }
-                      navigate(
-                        getTrialListLink(trial.study_id, excludedStates, next)
-                      )
-                    } else {
-                      navigate(
-                        getTrialListLink(trial.study_id, excludedStates, [
-                          trial.number,
-                        ])
-                      )
-                    }
-                  }}
-                  selected={
-                    selected.findIndex((t) => t.number === trial.number) !== -1
-                  }
+          <Box
+            sx={{
+              width: "100%",
+              height: `${rowVirtualizer.getTotalSize()}px`,
+              position: "relative",
+            }}
+          >
+            {rowVirtualizer.getVirtualItems().map((virtualItem) => {
+              console.log(virtualItem.index, virtualItem.start, virtualItem.end)
```

Comment:
[nits] Is it for debugging?

Reply:
Thank you for your comment.
I deleted this since it is for debugging.